src/System.IO.FileSystem/tests/Resources/System.IO.FileSystem.Tests.rd.xml

```diff
@@ -0,0 +1,6 @@
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
+    <!-- Needed because of FileOptions instance in [Theory] data which causes xunit to reflect on its ToString() -->
```

Comment:
qq, will this still be necessary once @morganbr checks in his Reflection improvements to better detect methods that should  be kept when compiling? I ask because if so, then we should link this to an issue so that we remove the workaround once that is in.